Library/Homebrew/formula.rb

```diff
@@ -1457,9 +1457,11 @@ def std_cmake_args
   end
 
   # Standard parameters for Go builds.
-  sig { returns(T::Array[T.any(String, Pathname)]) }
-  def std_go_args
-    ["-trimpath", "-o", bin/name]
+  sig { params(ldflags: T.nilable(String)).returns(T::Array[String]) }
+  def std_go_args(ldflags: nil)
+    args = ["-trimpath", "-o=#{bin/name}"]
```

Comment:
Could we also have an option to provide the name of the binary? It's not always the name of the formulae, especially for some formulae which install multiple executables.

Reply:
Dropping `-o` entirely would be the best way to do that, but it's a bit late for that now. We shouldn't really have variable things in "standard" args. `ldflags` is a bit of an exception due to restrictions in how passing multiple flags works.

Reply:
Yea, fair enough. I guess `-o` can be cleanly overridden by being passed later, unlike `-ldflags`.